content/shmem_bundle_start.tex

```diff
@@ -0,0 +1,35 @@
+\apisummary{
+    Start a communication bundle.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_bundle\_start}@(void);
+void @\FuncDecl{shmem\_ctx\_bundle\_start}@(shmem_ctx_t ctx);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{ctx}{A context handle specifying the context on which to
+    perform the optimization. When this argument is not provided, the
+    optimization is performed on the default context.}
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_bundle\_start} routine provides a hint to the \openshmem
+    library to being applying bundling-related optimizations.
+}
```

Comment:
Do we need to handle a scenario - when multiple starts are called recursively without breaking the bundle?
```c++
shmem_start_bundle(ctx1)
shmem_start_bundle(ctx1)
shmem_start_bundle(ctx1)
shmem_stop_bundle(ctx1)
```

Reply:
I'm not fond of nesting or disallowing.  Just a note that stop affects them all in that example?